runner.rb

```diff
@@ -10,4 +10,75 @@
   game_teams: game_teams_path
 }
 
-stat_tracker = StatTracker.from_csv(locations)
\ No newline at end of file
+stat_tracker = StatTracker.from_csv(locations)
+
+puts "GAME STATISTICS"
+puts "Highest Total Score:"
+p stat_tracker.highest_total_score
+puts
```

Comment:
I like the header and method name to console output. Keeps track of our data points.